sqlmesh/core/loader.py

```diff
@@ -57,15 +57,7 @@ def update_model_schemas(
         cache_hit = optimized_query_cache.with_optimized_query(model)
         schema.add_table(name, model.columns_to_types, dialect=model.dialect)
 
-        if any(dep not in models for dep in model.depends_on):
```

Comment:
do you think this is a good opportunity to educate users about external models?

Reply:
We should probably look for a different way, or just print a message instead of failing. Otherwise the platform's behavior is inconsistent: we fail if the target model can be rendered at parse time, but happily proceed if it can't. What do you think?

Reply:
yes i meant print a message